clippy_tests/examples/range.stderr

```diff
@@ -1,11 +1,3 @@
-error: Range::step_by(0) produces an infinite iterator. Consider using `std::iter::repeat()` instead
```

Comment:
@messense  `step_by` is no longer a Range method due to rust-lang/rust#42310. [These lines in `clippy_lints/src/ranges.rs`](https://github.com/Manishearth/rust-clippy/blob/master/clippy_lints/src/ranges.rs#L95-L97) should be modified.